### PR TITLE
feat(javascript): add Bunicorn

### PR DESCRIPTION
**Pull request by ragokan**
Hello, I wanted to add the new framework that is running on top of Bun. I was not able to test locally because I was not able to run it on Apple Silicon. I know, the framework yet does not have a documentation but I am creating it right now and wanted to see how it performs and see it on https://web-frameworks-benchmark.netlify.app. Thanks!

### CODE REVIEW

File: javascript/bunicorn/config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+framework:
+  website: https://github.com/ragokan/bunicorn
```
Comment (waghanza) [inline javascript/bunicorn/config.yaml:2]:
```suggestion
  github: ragokan/bunicorn
```

**Review by waghanza — APPROVED** [review]
Seems you succeed :tada:

**Comment by ragokan** [comment]
> Seems you succeed 🎉

When I merge, does it mean that it can be available on website soon? Btw, I added your suggestion, thank you!

**Comment by waghanza** [comment]
Results are update every monday (actually with some manual steps since on my workstation, with an arch closed to a cloud server)

**Comment by ragokan** [comment]
> Results are update every monday (actually with some manual steps since on my workstation, with an arch closed to a cloud server)

I understood it, thank you so much for response. Since I updated the code again, I think it needs a new approval. 
Have a great day!

**Pull request merged**